app/src/css/vendor/datetimepicker/_dtpoverrides.scss

```diff
@@ -49,4 +49,42 @@ styling overrides
 /* .react-datetime-picker__calendar .react-calendar, .react-datetime-picker__clock.react-datetime-picker__clock--open, .react-datetime-picker__calendar-button.react-datetime-picker__button {
   visibility: hidden;
   display: none;
-} */
\ No newline at end of file
+} */
+
+.react-datetime-picker__wrapper {
+  border: 1px solid lighten($lightest-grey, 4%) !important;
+  box-shadow: $shadow__default;
+  background-color: $white;
+  border-radius: .5em;
+}
+
+/*.react-datetime-picker__inputGroup__input option:first-child {
```

Comment:
I purposely commented it out for now because if there is still issues with alpha to numeric value spacing and the meridian selector (or absense of it) after you complete the other tickets, we may have to revisit this code.

Reply:
ok. thanks